### PR TITLE
💄 style : Convert some ico files from online to local.

### PR DESCRIPTION
**Pull request by RubuJam**
#### 💻 变更类型 | Change Type

- [ ] ✨ feat
- [ ] 🐛 fix
- [ ] ♻️ refactor
- [x] 💄 style
- [ ] 🔨 chore
- [ ] ⚡️ perf
- [ ] 📝 docs

#### 🔀 变更说明 | Description of Change
Convert some ico files from online to local.
将一些 ico 文件从在线访问转换为本地访问

#### 📝 补充信息 | Additional Information
When a managed challenge is set up(For example cloudflare managed challenge), the managed challenge web will display the icon.
当设置CloudFlare等服务商的托管质询后，托管质询网页会显示应用图标

### CODE REVIEW

File: src/app/metadata.ts [hunk 1]
```diff
@@ -22,11 +22,9 @@ const metadata: Metadata = {
   },
   description,
   icons: {
-    apple:
-      'https://registry.npmmirror.com/@lobehub/assets-favicons/latest/files/assets/apple-touch-icon.png',
-    icon: 'https://registry.npmmirror.com/@lobehub/assets-favicons/latest/files/assets/favicon-32x32.png',
-    shortcut:
-      'https://registry.npmmirror.com/@lobehub/assets-favicons/latest/files/assets/favicon.ico',
+    apple:'public/icons/apple-touch-icon.png',
```
Comment (arvinxx) [inline src/app/metadata.ts]: 如果是 public 的地址，不需要再额外加 `public/` 前缀了，直接 `/icons/apple-touch-icon.png`

Comment (RubuJam) [reply src/app/metadata.ts]: 谢谢你的提醒，我不知道这个知识，目前我已经修改代码，并且通过了测试

Thank you for your reminder, I didn't know this knowledge. I have modified the code and it passed the test.

File: src/app/metadata.ts [hunk 2]
```diff
@@ -22,11 +22,9 @@ const metadata: Metadata = {
...
   description,
   icons: {
-    apple:
-      'https://registry.npmmirror.com/@lobehub/assets-favicons/latest/files/assets/apple-touch-icon.png',
-    icon: 'https://registry.npmmirror.com/@lobehub/assets-favicons/latest/files/assets/favicon-32x32.png',
-    shortcut:
-      'https://registry.npmmirror.com/@lobehub/assets-favicons/latest/files/assets/favicon.ico',
+    apple:'icons/apple-touch-icon.png',
+    icon:'favicon.ico',
+    shortcut:'favicon-32x32.ico',
```
Comment (arvinxx) [inline src/app/metadata.ts:27]: shortcut 和icon 是不是反了

Comment (RubuJam) [reply src/app/metadata.ts:27]: 没写反，我查了一些资料，`shortcut`一般用于浏览器地址栏左侧显示的图标，比`icon`小

Comment (RubuJam) [reply src/app/metadata.ts:27]: 没写反，我查了一些资料并且做了一次测试，这里的`shortcut`用于浏览器地址栏左侧显示的图标，比`icon`小，原来的写法是错误的

It is not written backwards. I checked some information and did a test. The `shortcut` here is used for the icon displayed on the left side of the browser address bar. It's smaller than the `icon`. The original writing is wrong.

Comment (arvinxx) [reply src/app/metadata.ts:27]: 能否贴一下你查的资料？我学习一下~

Comment (RubuJam) [reply src/app/metadata.ts:27]: 我参考了[这篇文章](https://stackoverflow.com/questions/4014823/does-a-favicon-have-to-be-32×32-or-16×16)

另外我这么做的原因是，我尝试用其他图像指向`shortcut`（其余`favicon.ico`，`favicon-32x32.ico`均未改变，该项测试基于本次PR），发现仅有浏览器地址栏左侧显示的图标改变了，其余例如PWA应用图标都未改变。

于是我认为这里的`shortcut`仅用于浏览器地址栏左侧显示的图标。做出了更改


I referred to [this post](https://stackoverflow.com/questions/4014823/does-a-favicon-have-to-be-32×32-or-16×16).

Another reason I did this is that I tried to point other images to `shortcut` (the rest of `favicon.ico`, `favicon-32x32.ico`, etc. were not changed, this test was based on this PR), and found that only the icon displayed on the left side of the browser address bar changed, and the rest, such as the PWA application icon, did not change.

So I think the `shortcut` here is only used for the icon displayed on the left side of the browser address bar，Then made this modification.

**Review by arvinxx — APPROVED** [review]
Thanks for your Attention to the detail of project! I have learned something new from it 👍

**Pull request merged**